### PR TITLE
[Feature] Application snapshot resume section

### LINKED ISSUE #6928: ✨ Change experience component in Application Snapshot
## ✨ Feature

We have an inline experience component in the Application Snapshot that is not the best one to use for this ocassion. We should replace the experience section component for the one used in the "Review your resume" page during the application.

This allows the Pool operator to sort and order the resume section for easier browsing. 

## 🕵️ Details

The change should be done to the "Application Snapshot" page.
Replace current way of displaying Resume with different component.
Needed changes to the component:
1. Component should **NOT** have an "Add an experience" button
2. Experience items should **NOT** have an "Edit" button

<details><summary>This is how it currently looks</summary>
<p>

image

</p>
</details> 

<details><summary>This is how it should look</summary>
<p>

image

</p>
</details> 

[This is the component we SHOULD be using (Chromatic)](https://www.chromatic.com/component?appId=61e099a1bb1465003a73d3ce&csfId=pages-application-revamp-review-resume--has-experiences&buildNumber=5091&k=64875a9c83d25083ca10238d-1200-interactive-true&h=3&b=-2)

[This is the link to the Figma file](https://www.figma.com/file/BXYVsu1fo9LS2IySJDUINY/Candidates---Job-application---Version-1-to-6?type=design&node-id=636%3A54957&t=vMTI1A7jrahEP6Up-1)

## 🎨 Design File

Include a link to the design file (if it exists).

## 📸 Screenshot

Add a screenshot of the new feature (if possible).

## 🙋‍♀️ Proposed Implementation

(optional) Do you have a proposed implementation?

## 🌎 Localization

(optional) Provide any new copy along with translations available.

## ✅ Acceptance Criteria

A set of assumptions which, when tested, verify that the feature was properly implemented.

- [ ] In the resume section of the Admin Application Snapshot view, use the version of the experience list without skill "trees" and with sorting. (Same component as on the Resume section of the application, except without edit button.)

### PR DESCRIPTION
**Pull request by yonikid15**
🤖 Resolves #6928 

## 👋 Introduction

- Changes Resume section to use experience list without the tree view, on the application snapshot page.
- Adds a null state when no experiences are present (waiting on translations).

## 🧪 Testing

1. Login as admin and go to Pools table.
2. Select "View candidates" on a pool with candidates.
3. Select "View Application" on one of the pool candidates. Scroll down to Resume section.
4. Ensure section looks correct and all sorting works correctly.

### CODE REVIEW

File: apps/web/src/pages/PoolCandidates/ViewPoolCandidatePage/components/ResumeSection/index.ts
Comment (esizer) [inline apps/web/src/pages/PoolCandidates/ViewPoolCandidatePage/components/ResumeSection/index.ts:1]: We have an use to remove barrel exports like this so we should probably avoid introducing additional instances of it 😬

**Review by esizer — CHANGES_REQUESTED** [review]
Much better! 🧑‍🍳 Just one minor suggestion while we wait for translations 😄

File: apps/web/src/pages/PoolCandidates/ViewPoolCandidatePage/components/ResumeSection/index.ts
Comment (yonikid15) [reply apps/web/src/pages/PoolCandidates/ViewPoolCandidatePage/components/ResumeSection/index.ts:1]: Okay I think I got it with [d5c75fb](https://github.com/GCTC-NTGC/gc-digital-talent/pull/7075/commits/d5c75fb2efdb03ab63f2730ea5beac8e3021fbbd)

**Review by esizer — APPROVED** [review]
LGTM! 👍

**Pull request merged**